test/tests/errors.test.ts

```diff
@@ -11,20 +11,20 @@ function getFile(allFiles: any, path: string): string {
 
 it('Correctly errors when parsing faulty frontmatter', async () => {
 	const content = getFile(files, '/test/fixtures/errors/frontmatter.astro');
-	expect(() => format(content, {})).toThrowError('Expression expected');
+	expect(async () => await format(content, {})).toThrowError('Expression expected');
```

Comment:
I don't understand why this is failing, I will try later. Please @Princesseuh take a look at this when you can, no pressure!

Reply:
expect.toThrowError should work for async as well I think? maybe the expect needs to be awaited in this case???

Reply:
I just change it. I use [this](https://stackoverflow.com/questions/47144187/can-you-write-async-tests-that-expect-tothrow) example